config/crd/bases/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -2136,6 +2136,2129 @@ spec:
             type: object
         type: object
     served: true
+    storage: false
```

Comment:
You cannot change the storage version in the same release that you add a new version, else you will not be able to rollback. You should keep the v1alpha1 as the storage version for at least 1 release